src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -135,40 +126,34 @@ public WsdlProject importPostmanCollection(WorkspaceImpl workspace, String fileP
                     return null;
                 }
                 project.setDescription(postmanCollection.getDescription());
-                List<PostmanCollection.Request> requests = postmanCollection.getRequests();
-                for (PostmanCollection.Request request : requests) {
+                List<Request> requests = postmanCollection.getRequests();
+
+                SoapServiceCreator soapServiceCreator = new SoapServiceCreator(project);
```

Comment:
Maybe we should move the creation of this service into `if (request.isSoap()) ` block?
I don't see it being used anywhere else inside this method

Reply:
Good catch, I'll move it